releases/release-1.24/release-team.md

```diff
@@ -0,0 +1,17 @@
+# Kubernetes 1.23 Release Team
+
+| **Role** | **Name** (**GitHub / Slack ID**) | **Shadow Name(s) (GitHub / Slack ID)** |
+|----------|----------------------------------|----------------------------------------|
+| Lead | James Laverack ([@JamesLaverack](https://github.com/JamesLaverack) / Slack: `@james.laverack`) | |
+| Enhancements | | |
+| CI Signal | | |
+| Bug Triage | | |
+| Docs | | |
+| Release Notes | | |
+| Communications | | |
+| Emeritus Adviser | Joseph Sandoval ([@jrsapi](https://github.com/jrsapi) / Slack: `@Joseph`) | |
+| Branch Manager | | |
+
+Review the [Release Managers page](/release-managers.md) for up-to-date contact information on Release Engineering personnel.
+
+The schedule for all patch releases can be found at [Patch Releases page](/releases/patch-releases.md). It will be updated to include 1.23, once the 1.23 release cycle concludes.
```

Comment:
```suggestion
The schedule for all patch releases can be found at [Patch Releases page](/releases/patch-releases.md). It will be updated to include 1.24, once the 1.24 release cycle concludes.
```